install.sh

```
@@ -0,0 +1,287 @@
+    if [ "$actual_checksum" = "$expected_checksum" ]; then
+        print_info "Checksum verification passed"
+        return 0
+    else
+        print_error "Checksum verification failed"
+        return 1
+    fi
+}
+
+# Function to detect shell profile file
+detect_shell_profile() {
+    local shell_name=$(basename "$SHELL")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)